ogr/services/mock/github.py

```diff
@@ -0,0 +1,47 @@
+import github as github_origin
+import logging
+from ogr.mock_core import PersistentObjectStorage
+from typing import Type
+
+logger = logging.getLogger(__name__)
+
+old__requestEncode = github_origin.MainClass.Requester._Requester__requestEncode
+
+
+def new__requestEncode(self, cnx, verb, url, parameters, requestHeaders, input, encode):
+    internal_keys = [verb, url, parameters]
+    if self.persistent_storage.write_mode:
+        status, responseHeaders, output = old__requestEncode(
+            self, cnx, verb, url, parameters, requestHeaders, input, encode
+        )
+        self.persistent_storage.store(
+            keys=internal_keys, values=[status, responseHeaders, output]
```

Comment:
do we actually want to store headers? I'd probably just store the json payload; I'm worried that one of the headers is a session and you can get hacked easily

Reply:
I also though about that and I do not know if this header is used by some method or other classes, so I copied them because I think there are no personal data what could be misused.

I understand your opinion that it can be hacked for session I also though about that and tried to find any private secure data there, that there will be session key, but I think that this session key is valid just very short time and works just for this opened session and any other session later cannot use this key. maybe it could be used as MITM attack when current session is opened in case session is right now opened.  Maybe I'm wrong

Reply:
@TomasTomecek I also looked to headers and there should not be  any private data what can be used and I think these headers are used for creating objects so I should not remove them (are forwarded by methods like: requestJsonAndCheck what calls __check method and returns it)